### PR TITLE
ci: add sandbox

### PR DESCRIPTION
**Pull request by ypoplavs**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

File: .github/workflows/sandbox.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+permissions:
+  contents: write
+  id-token: write
+  pull-requests: write
+  issues: write
+
+
+env:
+  ALPINE_IMAGE: alpine:3.18.3
```
Comment (dejanzele) [inline .github/workflows/sandbox.yaml:19]: can we update that to v3.20.2?

File: .github/comment-template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Sandbox environment for `{{ .branch_ref }}` has been created.
```
Comment (dejanzele) [inline .github/comment-template.md:1]: I'd name the file `sandbox-comment-template.md` but I don't have a strong opinion

File: .github/workflows/sandbox.yaml (hunk 1 above)
Comment (ypoplavs) [reply .github/workflows/sandbox.yaml:19]: not in a scope of this task as it might bring breaking changes

Comment (ypoplavs) [reply .github/workflows/sandbox.yaml:19]: @vsukhin is that safe to upgrade?

Comment (vsukhin) [reply .github/workflows/sandbox.yaml:19]: It should be. But I would check it on dev

**Review by dejanzele — APPROVED** [review]

**Pull request merged**